lib/components/Toastr.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import * as R from "ramda";
 import { toast, Slide } from "react-toastify";
+import { hyphenize } from "../common";
```

Comment:
@datt _a This import is not used in the file. Please remove it. Rest looks good. 
Please make this change and the merge the PR. Thanks